kk/models/images.py

```diff
@@ -15,16 +12,15 @@ def get_images_dir():
 file_storage = FileSystemStorage(location=get_images_dir())
 
 
-class CommonImage(ModifiableModel):
+class Image(ModifiableModel):
     title = models.CharField(verbose_name=_('The title'), max_length=255, blank=True, default='')
     caption = models.TextField(verbose_name=_('Caption'), blank=True, default='')
     height = models.IntegerField(verbose_name=_('Height'), default=0)
     width = models.IntegerField(verbose_name=_('Width'), default=0)
     image = ThumbnailerImageField(verbose_name=_('Image'), storage=file_storage, resize_source=dict(size=(100, 100), sharpen=True))
 
+class WithImageMixin(models.Model):
+    images = models.ManyToManyField(Image)
```

Comment:
Using a ManyToManyField makes it impossible to stably order images, which I assume is a requirement.